modules/api/src/main/java/org/apache/ignite/table/mapper/OneColumnMapperImpl.java

```diff
@@ -26,6 +28,10 @@
  * @param <ObjectT> Target type.
  */
 class OneColumnMapperImpl<ObjectT> implements OneColumnMapper<ObjectT> {
+
+    /** Cache of one column mappers. */
+    private static final ConcurrentHashMap<CacheKey, OneColumnMapper<?>> CACHE = new ConcurrentHashMap<>();
```

Comment:
This cache allow us to reuse `OneColumnMapperImpl`objects.

So, instead of creating `OneColumnMapperImpl` - object of 3 fields, we create a `CacheKey` object of 2 fields and do synchronized lookup into a map (compute call).

What are we trying to fix here? Object constructor is trivial.

Reply:
Removed caching of mappers.